adyenv6backoffice/resources/adyenv6backoffice-backoffice-config.xml

```diff
@@ -24,9 +24,10 @@
 				<editorArea:section name="hmc.adyen.account">
 					<editorArea:attribute qualifier="adyenMerchantAccount"/>
 					<editorArea:attribute qualifier="adyenImmediateCapture"/>
-					<editorArea:attribute qualifier="adyenAllowedCards"/>
 					<editorArea:attribute qualifier="adyenRecurringContractMode"/>
 					<editorArea:attribute qualifier="adyenTestMode"/>
+					<editorArea:attribute qualifier="adyenGuestUserTokenisation"/>
```

Comment:
can you please change to "tokeni**z**ation" in other instances as well?
```suggestion
					<editorArea:attribute qualifier="adyenGuestUserTokenization"/>
```

Reply:
I was also confused then copied from magento ;) . Will change it :)